### PR TITLE
TB-5357 - Add infinite scroll to the section overview page for courses

### PR DESCRIPTION
**Pull request by sjoerdvandervis**
### HTT

* Checkout this branch and add a course with more then 10 sections;
* See that you get the load more which loads the next sections with Ajax.

### CODE REVIEW

File: social_course.install [hunk 1]
```diff
@@ -360,3 +360,24 @@ function social_course_update_8009() {
...
+      'views_infinite_scroll' => [
+        'button_text' => 'Load more',
+        'automatically_load_content' => FALSE,
+      ],
+    ]
+  ];
+  $display_options['use_ajax'] = TRUE;
+  // Set the display options in the config.
+  $view->set('display.default.display_options', $display_options)->save();
+}
```
Comment (navneet0693) [inline social_course.install:383]: New line!

File: config/install/views.view.course.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ dependencies:
     - group
     - node
     - user
+    - views_infinite_scroll
```
Comment (navneet0693) [inline config/install/views.view.course.yml:11]: It's a good idea to add dependency here: https://github.com/goalgorilla/social_course/blob/8.x-2.x/social_course.info.yml and https://github.com/goalgorilla/social_course/blob/8.x-2.x/composer.json

**Review by navneet0693 — CHANGES_REQUESTED** [review]

File: config/install/views.view.course.yml (hunk 2 above)
Comment (sjoerdvandervis) [reply config/install/views.view.course.yml:11]: Added!

File: social_course.install (hunk 1 above)
Comment (sjoerdvandervis) [reply social_course.install]: Done!

**Review by navneet0693 — APPROVED** [review]

**Comment by navneet0693** [comment]
We will not be merging this until all the stakeholders agree to introduce this feature in our module.

File: social_course.install [hunk 3]
```diff
@@ -360,3 +361,45 @@ function social_course_update_8009() {
...
+    if (is_file($config_file)) {
+      $settings = Yaml::parse(file_get_contents($config_file));
+      if (is_array($settings)) {
+        $config = \Drupal::configFactory()
+          ->getEditable($key);
+        $config->setData($settings)->save(TRUE);
+      }
+    }
+  }
+}
```
Comment (navneet0693) [inline social_course.install:405]: New line :D

**Review by ribel — APPROVED** [review]
I have tested the changes and all works good, so I will include it the next release.

**Pull request merged**